src/DefaultEmissionManager.sol

```diff
@@ -62,7 +70,7 @@ contract DefaultEmissionManager is Ownable2StepUpgradeable, IDefaultEmissionMana
         uint256 amountToMint = newSupply - currentSupply;
         if (amountToMint == 0) return; // no minting required
 
-        uint256 treasuryAmt = amountToMint / 3;
+        uint256 treasuryAmt = amountToMint * 2 / 5;
```

Comment:
Moving emission from 1/3 to 2/5 is not properly explained for Treasury, it is recommended to improve documentation in order anyone can follow the math easily. It seems that 2/5 allow us to keep a 0.4 (40%) from all minted POL while StakeManager always takes 1.5 times Treasury funds. Please confirm.

Reply:
added a comment in [6ebaba5](https://github.com/0xPolygon/pol-token/pull/58/commits/6ebaba579efc96995ce3cc661adaffb91b31a106)